tests/test_inherits_path_option.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+
+from twitter.common.contextutil import temporary_dir
+
+from pex.pex_builder import PEXBuilder
+from pex.testing import run_simple_pex, write_simple_pex
+
+
+def write_inheriting_pex(td, exe_contents, dists=None, coverage=False):
+  """Write a pex file that contains an executable entry point
+
+  :param td: temporary directory path
+  :param exe_contents: entry point python file
+  :type exe_contents: string
+  :param dists: distributions to include, typically sdists or bdists
+  :param coverage: include coverage header
+  """
+  dists = dists or []
+
+  with open(os.path.join(td, 'exe.py'), 'w') as fp:
+    fp.write(exe_contents)
+
+  pb = PEXBuilder(path=td, preamble=COVERAGE_PREAMBLE if coverage else None)
```

Comment:
This line doesn't fail only because coverage is never passed an defaults to `False`, otherwise boom on `COVERAGE_PREAMBLE` which is not a symbol in-scope.  Killing the unused `dists` and `coverage` params would be good.